packages/jest-utils/src/jest-utils.types.ts

```diff
@@ -1,22 +1,23 @@
 export interface EmpathyExtendedExpect {
   arrayOf(classType: Newable): any;
-  arrayOfItemsMatching(schema: {}): any;
+  arrayOfItemsMatching(schema: Record<string, any>): any;
```

Comment:
Just for my information, did the linter complain about this?

Reply:
the linter doesn't like to use an empty object, so i added a record